test/mechs/sandiego.yaml

```diff
@@ -0,0 +1,225 @@
+generator: cti2yaml
```

Comment:
Given that we now have the `.yaml` versions, do we still need the `.cti` versions?

Reply:
It might be nice to have one just to test that they are still supported?

Reply:
Fair point. I don't see a good reason to keep all of them in both formats though.